tools/deviceGen/input/device_limits.json

```diff
@@ -1,2339 +1,2253 @@
+                    "name": "sampledImageIntegerSampleCounts",
+                    "value": "NoMSor4Samples",
+                    "expose": "DISABLE"
+                },
+                {
+                    "type": "core::bitflag<asset::IImage::E_SAMPLE_COUNT_FLAGS>",
+                    "name": "sampledImageDepthSampleCounts",
+                    "value": "NoMSor4Samples",
+                    "expose": "DISABLE"
+                },
+                {
+                    "type": "core::bitflag<asset::IImage::E_SAMPLE_COUNT_FLAGS>",
+                    "name": "sampledImageStencilSampleCounts",
+                    "value": "NoMSor4Samples",
+                    "expose": "DISABLE"
+                },
+                {
+                    "type": "core::bitflag<asset::IImage::E_SAMPLE_COUNT_FLAGS>",
+                    "name": "storageImageSampleCounts",
+                    "value": "asset::IImage::E_SAMPLE_COUNT_FLAGS::ESCF_1_BIT",
+                    "expose": "DISABLE"
+                }
             ]
         },
-        {
-            "type": "core::bitflag<asset::IImage::E_SAMPLE_COUNT_FLAGS>",
-            "name": "sampledImageColorSampleCounts",
-            "value": "NoMSor4Samples",
-            "expose": false
-        },
-        {
-            "type": "core::bitflag<asset::IImage::E_SAMPLE_COUNT_FLAGS>",
-            "name": "sampledImageIntegerSampleCounts",
-            "value": "NoMSor4Samples",
-            "expose": false
-        },
-        {
-            "type": "core::bitflag<asset::IImage::E_SAMPLE_COUNT_FLAGS>",
-            "name": "sampledImageDepthSampleCounts",
-            "value": "NoMSor4Samples",
-            "expose": false
-        },
-        {
-            "type": "core::bitflag<asset::IImage::E_SAMPLE_COUNT_FLAGS>",
-            "name": "sampledImageStencilSampleCounts",
-            "value": "NoMSor4Samples",
-            "expose": false
-        },
-        {
-            "type": "core::bitflag<asset::IImage::E_SAMPLE_COUNT_FLAGS>",
-            "name": "storageImageSampleCounts",
-            "value": "asset::IImage::E_SAMPLE_COUNT_FLAGS::ESCF_1_BIT",
-            "expose": false
-        },
         {
             "type": "uint8_t",
             "name": "maxSampleMaskWords",
             "value": 1
         },
         {
-            "groupComment": [
-                "[REQUIRE] ROADMAP 2024 and good device support"
+            "comment": [
+                "ROADMAP 2024 and good device support"
+            ],
+            "entries": [
+                {
+                    "type": "bool",
+                    "name": "timestampComputeAndGraphics",
+                    "value": true,
+                    "expose": "DISABLE"
+                },
+                {
+                    "type": "float",
+                    "name": "timestampPeriodInNanoSeconds",
+                    "value": 83.334,
+                    "compare": "REVERSE | DISABLE",
```

Comment:
huh? it should be just `DISABLE` and shouldn't pass your validation of inputs in the python script

Reply:
yes thats fine,

Reply:
why is it still `REVERSE | DISABLE` !?
 should only be `DISABLE` and a field with two things at once shouldn't pass your python validation!